### PR TITLE
framework for the css

### PR DESCRIPTION
**Pull request by ashwoodaa**
color scheme is all black until I pick an actual color scheme.

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+
+nav li {
+  display: inline-block;
+ }
+
+
+nav a {
+  font-weight: 800; /*Can you say BOLD?*/
+  padding: 15px 10px; /* yay, space. It was getting crowded and weird*/
+}
```
Comment (aoswalt) [inline stylesheet.css:102]: Add display: inline-block to make entire anchor clickable.

File: stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+  }
+
+  #secondary {
+    width: 30%;
+    float: right;
+  }
+
+  #third {
+    width: 30%;
+    float: ;
```
Comment (aoswalt) [inline stylesheet.css]: Empty property should be removed.

File: stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+things are for and such.
+
+Color scheme: I set everything to black because I needed place holders. Except for
+the body background and the nav links;
+*********************************************************************************/
+
+/********************************************************************************
+GENERAL
+*********************************************************************************/
+
```
Comment (aoswalt) [inline stylesheet.css:16]: Suggest adding box-sizing: border-box to all for less fragile sizing.
- {
  box-sizing: border-box;
  }

File: stylesheet.css (hunk 1 above)
Comment (ashwoodaa) [inline stylesheet.css:102]: :heavy_check_mark:

File: stylesheet.css (hunk 2 above)
Comment (ashwoodaa) [inline stylesheet.css]: :heavy_check_mark:

File: stylesheet.css (hunk 3 above)
Comment (ashwoodaa) [inline stylesheet.css:16]: :heavy_check_mark:

**Comment by ashwoodaa** [comment]
Changed em to rem

**Comment by tekishahammock** [comment]
If the frame has been finished, I'm cool with merging it in.

File: stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+ cupcakes which means when the economy tanks we are expendable af and will
+ be the first to go if our site isn't easy to use */
+
+h3 {
+  margin: 0 0 1rem 0; /*insurance against the h3 heading to be separated from the paragraphs*/
+}
+
+* {
+  box-sizing: border-box;
+}
```
Comment (aoswalt) [inline stylesheet.css]: Move the universal selector to the top for cascading overriding.

**Comment by aoswalt** [comment]
:+1:

**Comment by ashwoodaa** [comment]
Also moved the box sizing: border box to the top

**Pull request merged**